test/unit/ripple/index.test.js

```diff
@@ -272,3 +272,12 @@ test('throws error if no foundation', () => {
   const DivNoRefRipple = withRipple(DivNoRef);
   assert.throws(DivNoRefRipple.prototype.componentDidMount);
 });
+
+test('doesn\'t call #Component.setState after unmount', () => {
```

Comment:
If I comment out 
```js
// this.isMounted_ = false;
```
in the componentWillUnmount method, the tests still passes...so effectively this is not testing anything. Looking at a fix.

Reply:
I'm convinced there is no way to write a test for this with enzyme. Once you call unmount, you cannot get `wrapper.instance()` to check whether its true or false. I added tests to make sure that addClass/removeClass/updateCssVars methods do not update classLists/cssVars if `isMounted_` is false.